presto-native-execution/presto_cpp/main/tests/QueryContextManagerTest.cpp

```diff
@@ -31,16 +31,31 @@ TEST_F(QueryContextManagerTest, nativeSessionProperties) {
       .systemProperties = {
           {"native_max_spill_level", "1"},
           {"native_spill_compression_codec", "NONE"},
-          {"native_join_spill_enabled", "true"},
```

Comment:
Should we move this to another PR?

Reply:
This part is just expanding the unit testing and not changing any defaults so is fairly benign to keep in this PR. Please let me know if you still feel strongly about separating this.

Reply:
This is fine. Thanks for clarifying!